update_tag_sample_page.py

```diff
@@ -0,0 +1,50 @@
+import sys
+import argparse
+from rest import McM
+sys.path.append('/afs/cern.ch/cms/PPD/PdmV/tools/McM/')
+
+parser = argparse.ArgumentParser(description='Tag samples for an analysis subgroup')
+parser.add_argument('--input_file', default='text_not_existing.txt', required=True,
+                    help='input file where miniAOD names are stored')
+parser.add_argument('--tag', default='default_tag', required=True,
+                    help='tag to include in McM for the specified samples')
+parser.add_argument('--dev', default=True, required=False,
```

Comment:
Maybe this should be False by default? Also, add `action='store_true'` which just sets 'dev' to true if `--dev` is present and keeps it false if `--dev` is not there.